src/NuGet.Core/NuGet.Test.Utility/CommandRunnerResult.cs

```diff
@@ -4,20 +4,20 @@ namespace NuGet.Test.Utility
 {
     public class CommandRunnerResult
     {
-        internal CommandRunnerResult(Process process, int exitCode, LockedStringBuilder output, LockedStringBuilder error)
+        private readonly string _output;
+        private readonly string _error;
+
+        internal CommandRunnerResult(Process process, int exitCode, string output, string error)
         {
             Process = process;
             Item1 = exitCode;
             _output = output;
             _error = error;
         }
 
-        private readonly LockedStringBuilder _output;
-        private readonly LockedStringBuilder _error;
-
         public Process Process { get; }
         public int Item1 { get; }
-        public string Item2 => _output.ToString();
-        public string Item3 => _error.ToString();
+        public string Item2 => _output;
```

Comment:
I think this should be a different PR. This will change many, many files and crowd out the intent of this PR.